spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
This is getting out-of-proportion energy at this point. I've dropped the test. And I'll create PRs to remove other similar tests that already exist in our code base when I see them.